src/com/magento/idea/magento2plugin/actions/generation/dialog/AbstractDialog.java

```diff
@@ -119,7 +119,7 @@ protected void clearValidationHighlighting() {
      * @return List[FieldValidationData]
      */
     protected List<FieldValidationData> getFieldsToValidate() {
-        return new LinkedList<>(fieldsValidationsList);
+        return getAvailableForFiltering(fieldsValidationsList);
```

Comment:
Is there any reason to create a new method? Please move the `getAvailableForFiltering` implementation into this method, so we won't have redundant methods that do the same.

Reply:
@eduard13 Removed redundant method.